mvp-spec/audit-log-design.md

```diff
@@ -0,0 +1,257 @@
+        "source": {
+          "domain": "ad-server.com",
+          "timestamp": 1639589531,
+          "signature": "12345_signature"
+        }
+    },
+    "transmissions": [
+        {
+            "version": 0,
+            "receiver": "ssp1.com",
+            "status": "success",
+            "details": "",
+            "source": {
+                "domain": "ssp1.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        },
+        {
+            "version": 0,
+            "receiver": "ssp2.com",
+            "status": "success",
+            "details": "",
+            "source": {
+                "domain": "ssp2.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        },
+        {
+            "version": 0,
+            "receiver": "dsp.com",
+            "status": "success",
+            "details": "",
+            "source": {
+                "domain": "dps.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        }
+    ]
+}
+```
+<!--partial-end-->
+
+# Display of the Audit UI
+
+## Web Page interface
+
+Once the user has clicked on the Audit Button, she/he is redirected to the
+Audit UI. The Contracting Party generates the UI based on the validations 
+of the Prebid SSO Data and the Transmission Results. The following elements must
+appear in the page:
+
+| Element                          | Details                                   |
+|----------------------------------|-------------------------------------------|
+| List of Pseudonymous-Identifiers | Each Pseudonymous-Identifier must be paired with the name of the Operator who generated it and signed it with a Reg/Green indicator expressing the validity of the signature.                                                |
+| List of the Preferences          | Each Preference must be paired with the name of the CMP who generated it and signed it with a Red/Green indicator expressing the validity of the signature.                                                             |
+| Seed                             | The Seed is represented by the Transaction ID and a Red/Green indicator expressing the validity of the signature                                                                                                             |
+| List of Transmission Results     | The Transmission Results available in the Audit Log. Each Transmission Results is represented by the Name of the Receiver, the status of the Transmission and a Red/Green indicator expressing the validity of the signature |
+
+The design of the page is up to the Contracting Party but it must consider 
+to have a neutral UI to be used for many different Publishers in case of a DSP.
+
+For the version v0.1, there are only one possible preference
+("opt-in") and one possible pseudonymous-identifiers ("prebid_id").
```

Comment:
`use_browsing_for_personalization` and `paf_browser_id` (probably deserves a dedicated PR)

Reply:
We already discussed this: https://github.com/criteo/addressable-network-proposals/issues/59#issuecomment-1023292707